documentation/source/user_section/tutorials/shape-analysis/compute-csa-and-other-shape-metrics/other-shape-metrics.rst

```diff
@@ -3,11 +3,23 @@
 Other shape metrics
 ###################
 
-The csv files generated by ``sct_process_segmentation`` also include metrics to analyse the shape of the spinal cord in the axial plane, such as ellipticity, antero-posterior and right-left dimensions. These are of particular interest for studying cord compression. See `Martin et al. BMJ Open 2018 <https://bmjopen.bmj.com/content/8/4/e019809>`_ for an example application in degenerative cervical myelopathy.
+The csv files generated by ``sct_process_segmentation`` include besides the cross-sectional area (CSA) also the following metrics to analyse the shape of the spinal cord in the axial plane:
+
+- antero-posterior diameter (AP diameter), ``diameter_AP``
+- transverse diameter, ``diameter_RL``
+- eccentricity, ``eccentricity``
+- solidity, ``solidity``
+- orientation, ``orientation``
+
+These are of particular interest for studying cord compression. See `Martin et al. BMJ Open 2018 <https://bmjopen.bmj.com/content/8/4/e019809>`_ and `Horáková et al. Quant Imaging Med Surg 2022 <https://pubmed.ncbi.nlm.nih.gov/35371944/>`_ for an example applications.
+
+For an interactive database of normative values, see `Valošek, Bédard et al. NeuroLibre 2023 <https://neurolibre.org/papers/10.55458/neurolibre.00017>`_.
 
 .. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/shape-metric-computation/sct_process_segmentation-shape-metrics.png
    :align: center
 
+   Figure Caption: (A) Compression at the level of the intervertebral disc C5-C6 (red arrows) is manifested by (B) the decrease of AP Diameter, Compression Ratio, and Solidity and by the increase of Eccentricity (red arrows). The blue line represents normative values computed from 105 males from the spine-generic dataset. Adapted from `Valošek, Bédard et al. NeuroLibre 2023 <https://doi.org/10.55458/neurolibre.00017>`_.
```

Comment:
```
other-shape-metrics.rst:18: WARNING: Figure caption must be a paragraph or empty comment.

.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/shape-metric-computation/sct_process_segmentation-shape-metrics.png
   :align: center

   Figure Caption: (A) Compression at the level of the intervertebral disc C5-C6 (red arrows) is manifested by (B) the decrease of AP Diameter, Compression Ratio, and Solidity and by the increase of Eccentricity (red arrows). The blue line represents normative values computed from 105 males from the spine-generic dataset. Adapted from `Valošek, Bédard et al. NeuroLibre 2023 <https://doi.org/10.55458/neurolibre.00017>`_.
```

Reply:
The figure caption formatting seems to be broken due to special characters in `Valošek, Bédard` names.

`Valosek, Bedard` (without special characters):

<img width="400" alt="image" src="https://github.com/spinalcordtoolbox/spinalcordtoolbox/assets/39456460/791c9c1f-1411-4e09-b2cc-ce85f1bddceb">

`Valošek, Bédard` (current version):

<img width="400" alt="image" src="https://github.com/spinalcordtoolbox/spinalcordtoolbox/assets/39456460/f76fd8cd-2e80-4840-9238-d387cea3e8bc">

Reply:
It turns out that it's not because of the special characters, but the fact that there is a duplicate link with the same name.

((If both links use special characters, it breaks. If both links use `e` and `s`, it breaks. If the links use different characters, it works.))

The real solution here is to [use anonymous references](https://github.com/sphinx-doc/sphinx/issues/3921#issuecomment-315581557).